code/gallery.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Gallery</title>
+    <link href="style.css" rel = "stylesheet" type="text/css"> 
+
+  </head>
+<body>
+  <h1>Gallery!</h1>
+    <div style="text-align: center"><img src="/Users/livartda/Documents/project-p1-first-webpage-master/code/SAM_1220.JPG" width="800" height="550"></div>
```

Comment:
- Don't forget indentation. Take a look at our [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) to learn more.
- Keep the styling to the CSS file. If you want to style elements different from each other you can give them class names
- Write relative paths to your assets: `src="./SAM_1220.JPG"`

**HTML example**
```
<div class="image-container">
  <img src="./SAM_1220.JPG">
</div>
```

**CSS example**
```
.image-container {
  text-align: center;
}
```

Reply:
Thank you for the guiding!